src/odemis/odemisd/test/mdupdater_test.py

```diff
@@ -0,0 +1,86 @@
+"""
+Created on 22 Feb 2022
+
+@author: Éric Piel
+
+Copyright © 2022 Éric Piel, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms 
+of the GNU General Public License version 2 as published by the Free Software 
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; 
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR 
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with 
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import numpy
+from odemis import model
+from odemis.driver import static
+from odemis.model import Microscope
+from odemis.odemisd.mdupdater import MetadataUpdater
+from odemis.util import mock
+import unittest
+import unittest.mock
+
+
+class MDUpdaterTest(unittest.TestCase):
+
+    def test_lens_observer(self):
+        # Create microscope (.Alive)
+        mic = Microscope("Fake mic", "secom")
+
+        # Create a CCD (the role matters)
+        img = model.DataArray(numpy.empty((512, 768), dtype=numpy.uint16))
+        ccd = mock.FakeCCD(img)  # role is always "ccd"
+
+        # Create Lens affects CCD (the role matters)
+        lens = static.OpticalLens("Lens", "lens", mag=0.51, pole_pos=(458, 519), focus_dist=0.5e-3,
+                                  mirror_pos_top=[600.5, 0.2], mirror_pos_bottom=(-200, 0.3))
```

Comment:
```suggestion
                                  mirror_pos_top=(600.5, 0.2), mirror_pos_bottom=(-200, 0.3))
```

Reply:
This is on purpose, because when reading a YAML file, all the "tuples" are lists. So we need to make sure passing a list works fine. I'll add a comment.